apps/core/src/hooks/useGetAllBalances.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '../api/RpcClientContext';
+import { type SuiAddress } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+
+export function useGetAllBalances(
+    address?: SuiAddress | null,
+    refetchInterval?: number,
+    staleTime?: number
+) {
+    const rpc = useRpcClient();
+    return useQuery({
+        queryKey: ['get-all-balance', address, staleTime, refetchInterval],
```

Comment:
Is it necessary to add `staleTime` and `refetchInterval` in the query key? Does it affect the data?